src/JustSaying/AwsTools/MessageHandling/SqsPolicyBuilder.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Amazon;
+
+namespace JustSaying.AwsTools.MessageHandling;
+
+internal static class SqsPolicyBuilder{
+
+    public static string BuildPolicyJson(SqsPolicyDetails policyDetails)
```

Comment:
Similar to @adammorr's request on #1082, could this be made an extension method for SqsPolicyDetails?

Reply:
@hwoodiwiss I've updated https://github.com/justeat/JustSaying/pull/1082 to make them both extension methods.
I've left this PR to just introduce the approval tests and seperation of the policy generation

Reply:
Great stuff!